src/message/encode.ts

```diff
@@ -56,17 +55,18 @@ export function encodePingMessage(m: IPingMessage): Buffer {
 }
 
 export function encodePongMessage(m: IPongMessage): Buffer {
-  const ipMultiaddr = multiaddr(`/${isIPv4(m.recipientIp) ? "ip4" : "ip6"}/${m.recipientIp}`);
-  const tuple = ipMultiaddr.tuples()[0][1];
-  if (!tuple) {
-    throw new Error("invalid address for encoding");
-  }
-  if (m.recipientPort < 0 || m.recipientPort > 65535) {
+  if (m.addr.port < 0 || m.addr.port > 65535) {
     throw new Error("invalid port for encoding");
   }
   return Buffer.concat([
     Buffer.from([MessageType.PONG]),
-    RLP.encode([toBuffer(m.id), toBuffer(m.enrSeq), tuple, m.recipientPort]),
+    RLP.encode([
+      //
```

Comment:
some comments here?

Reply:
Added this to force prettier into placing the arguments vertically